catroid/src/main/res/values/strings.xml

```diff
@@ -1577,5 +1577,6 @@
     <string name="crash_dialog_always_send">Always send</string>
     <string name="crash_dialog_send">Send</string>
     <string name="plus" translatable="false">+</string>
+    <string name="com.crashlytics.android.build_id">RANDOM_UUID</string>
```

Comment:
An exception was getting raised during the test execution and no tests were being performed.

`Test running failed: Instrumentation run failed due to 'io.fabric.sdk.android.services.concurrency.UnmetDependencyException'`

So I looked out for the last solution here and it worked:- [solution](https://stackoverflow.com/questions/28762624/com-crashlytics-android-crashlyticsmissingdependencyexception-during-gradle-tes )